tutorials/best_practices/godot_notifications.rst

```diff
@@ -102,6 +102,27 @@ implementing a Timer-timeout loop is another option.
         }
     }
 
+ .. code-tab:: cpp C++
+
+    using namespace godot;
+
+    class MyNode : public Node {
+    public:
+        // Allows for recurring operations that don't trigger script logic
+        // every frame (or even every fixed frame).
+        void _ready() override {
+            Timer *timer = memnew(Timer());
+            timer->set_autostart(true);
+            timer->set_wait_time(0.5);
+            add_child(timer);
+
+            // I have no idea what this would be in C++  :(
```

Comment:
You should add a method directly, as we don't have a callable for lambdas in c++